bundles/aliyun-bundle/build.gradle.kts

```diff
@@ -25,8 +25,15 @@ plugins {
 }
 
 dependencies {
+  compileOnly(project(":api"))
+  compileOnly(project(":core"))
+  compileOnly(project(":catalogs:catalog-common"))
   compileOnly(project(":catalogs:catalog-hadoop"))
   compileOnly(libs.hadoop3.common)
+
+  implementation(libs.aliyun.credentials.sdk)
+  implementation(libs.sun.activation)
```

Comment:
why add this dependences?

Reply:
Credentials Java SDK depends on this package, and JDK>9 requires manual add

Reply:
got it, thx